src/inject/dynamic-theme/css-rules.ts

```diff
@@ -6,8 +6,17 @@ import {logWarn} from '../utils/log';
 export function iterateCSSRules(rules: CSSRuleList, iterate: (rule: CSSStyleRule) => void) {
     forEach(rules, (rule) => {
         if (rule instanceof CSSMediaRule) {
-            const media = Array.from(rule.media);
-            if (media.includes('screen') || media.includes('all') || !(media.includes('print') || media.includes('speech'))) {
+            const media: string[] = [];
+            Array.from(rule.media).forEach((mediaRule) => {
+                if (mediaRule.includes(' and ')) {
+                    mediaRule.split(' and ').forEach((splittedRule) => media.push(splittedRule));
```

Comment:
Not sure, but I think it cannot be `(min-width: 10rem) and screen`. If it can, let's use `includes()` instead of `startsWith()`.

Reply:
`print, screen and (max-width: 64em)` will return an array `['print', 'screen and (max-width: 64em)']`. Just tried `print, screen and (max-width: 64em)`, it returned `['print', 'not all']`, simply saying it is not possible. So I think you can use `startsWith()` for sure.

Reply:
For the current sites that has this problem they use the same ways as this so I don't see an problem as well by using `startWith()`